javascript/builtins/SharedArrayBuffer.json

```
@@ -16,11 +16,17 @@
                 "notes": "Chrome disabled SharedArrayBuffer on January 5, 2018 to help reduce the efficacy of <a href='https://www.chromium.org/Home/chromium-security/ssca'>speculative side-channel attacks</a>. This was a temporary removal while mitigations were put in place."
               }
             ],
-            "chrome_android": {
-              "version_added": "60",
-              "version_removed": "63",
-              "notes": "Chrome disabled SharedArrayBuffer on January 5, 2018 to help reduce the efficacy of <a href='https://www.chromium.org/Home/chromium-security/ssca'>speculative side-channel attacks</a>. This is intended as a temporary measure until other mitigations are in place."
-            },
+            "chrome_android": [
+              {
+                "version_added": "89",
+                "notes": "SharedArrayBuffer are available on Android gated behind COOP/COEP. Additional details can be found at <a href='https://web.dev/coop-coep/'>web.dev</a>"
```

A few copy editing suggestions here:

- Since this is attached to the Android data specifically, we don't have to mention Android specifically.
- Rather than mentioning the domain alone, I thought it might be nice to mention the title of the article instead
- Minor formatting and punctuation changes

What do you think?

```suggestion
                "notes": "<code>SharedArrayBuffer</code> is gated behind COOP/COEP. For more detail, read <a href='https://web.dev/coop-coep/'>Making your website "cross-origin isolated" using COOP and COEP</a>."
```